libobs/obs-internal.h

```diff
@@ -1080,6 +1080,9 @@ struct obs_encoder {
 	uint32_t timebase_num;
 	uint32_t timebase_den;
 
+	/* set before encoding is performed */
+	uint64_t cur_timestamp;
```

Comment:
ideally, this would not sit in `struct obs_encoder` but in `struct encoder_frame` and would also be forwarded somehow for textures ... however, putting this here does not break the ABI (as I understand it) ... I guess this could + should be moved on a major bump